### PR TITLE
Small fixes for OD skimmer

### PR DESCRIPTION
**Pull request by nikolayilyin**
- Better error message in case when an exception message is null.
- Default values for observations and iterations values when reading OD skims.

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3444)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
12:38:31 A problem occurred configuring project ':metrics2.0'.
12:38:31 > Could not resolve all artifacts for configuration ':metrics2.0:classpath'.
12:38:31    > Could not download snakeyaml-1.24.jar (org.yaml:snakeyaml:1.24)
12:38:31       > Could not get resource 'https://jcenter.bintray.com/org/yaml/snakeyaml/1.24/snakeyaml-1.24.jar'.
12:38:31          > Could not GET 'https://jcenter.bintray.com/org/yaml/snakeyaml/1.24/snakeyaml-1.24.jar'. Received status code 502 from server: Bad Gateway

**Comment by nikolayilyin** [comment]
test!

File: src/main/scala/beam/router/skim/CsvSkimReader.scala [hunk 1]
```diff
@@ -38,15 +36,15 @@ class CsvSkimReader(
...
+          logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath'. $ex")
           Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
       }
     }
   }
 
   def readSkims(reader: BufferedReader): Map[AbstractSkimmerKey, AbstractSkimmerInternal] = {
     tryReadSkims(reader).recover { case ex: Throwable =>
-      logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath': ${ex.getMessage}")
+      logger.warn(s"Could not read warmStart skim from '$aggregatedSkimsFilePath'. $ex")
```
Comment (dimaopen) [inline src/main/scala/beam/router/skim/CsvSkimReader.scala]: We could use the logger's feature to print the whole stack trace if we provide the exception as a parameter to `warn` method:
`, ex)`

File: src/main/scala/beam/router/skim/core/ODSkimmer.scala [hunk 2]
```diff
@@ -373,8 +381,8 @@ object ODSkimmer extends LazyLogging {
         cost = row("cost").toDouble,
         energy = Option(row("energy")).map(_.toDouble).getOrElse(0.0),
         level4CavTravelTimeScalingFactor = row.get("level4CavTravelTimeScalingFactor").map(_.toDouble).getOrElse(1.0),
-        observations = row("observations").toInt,
-        iterations = row("iterations").toInt
+        observations = toIntOrDefaultIfNull(row("observations"), 0),
```
Comment (dimaopen) [inline src/main/scala/beam/router/skim/core/ODSkimmer.scala]: Maybe we could use `org.apache.commons.lang3.math.NumberUtils#toInt(java.lang.String, int)` from apache commons-lang?

**Review by dimaopen — APPROVED** [review]

**Comment by nikolayilyin** [comment]
test!

**Pull request merged**